DcCore/DcCore/Helper/KeychainManager.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+import Security
+
+public enum KeychainError: Error {
+    case noPassword
+    case unhandledError(message: String, status: OSStatus)
+}
+
+public class KeychainManager {
+    private typealias KcM = KeychainManager
+    private static let teamId = "8Y86453UA8"
```

Comment:
We can only access the AppIdentifierPrefix if we manually add an key value pair in Info.plist

`<key>AppIdentifierPrefix</key>
<string>$(AppIdentifierPrefix)</string>` 

During build time the teamId would be written into the plist entry and on runtime we could access the value by calling 
`let appIdentifierPrefix =
    Bundle.main.infoDictionary!["AppIdentifierPrefix"] as! String
`

https://stackoverflow.com/questions/11726672/access-app-identifier-prefix-programmatically/28714850

Thumbs up, if you prefer that. Otherwise I would add the comment instead.

Reply:
we can leave it as it is now, now problem. but maybe add a comment, just similar to what you've just explained, maybe including the link.